stock_location_content_template/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Open Source Integrators (https://www.opensourceintegrators.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Location Content Template",
+    "summary": "Define templates of location content and check the report for discrepancies",
+    "version": "14.0.1.0.0",
+    "category": "stock",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "maintainers": ["max3903"],
+    "license": "AGPL-3",
+    "depends": ["stock", "sale_management"],
```

Comment:
```suggestion
    "depends": ["stock"],
```

I don't see any sale dependency here